airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -22,7 +22,7 @@
 from datetime import datetime
 from operator import attrgetter
 from time import time
-from typing import List, Optional, Tuple, Union
+from typing import Hashable, List, Optional, Tuple, Union
```

Comment:
Use `collections.abc.Hashable` instead (I believe that’s the canonical import)

Reply:
Done: 27aa8e3c8c1ddd5a6b22d562d1ae1006c51b1ded